_includes/svg/icon-close.svg

```diff
@@ -0,0 +1 @@
+<svg width="16" height="16" xmlns="http://www.w3.org/2000/svg"><path class="icon__close" d="M14.378 16L16 14.378 9.622 8 16 1.622 14.378 0 8 6.378 1.622 0 0 1.622 6.378 8 0 14.378 1.622 16 8 9.622z" fill-rule="nonzero"/></svg>
```

Comment:
I ran this through SVGO to get rid of a bunch of silly bits of code that Sketch adds. I used this UI to do it: https://jakearchibald.github.io/svgomg/

It also helps to compress the image, and reduce the complexity of the paths on the svg. It's good stuff! :smile: